wiki/Article_Styling_Criteria/Images/en.md

```diff
@@ -0,0 +1,92 @@
+# Images
+
+_Main page: [Article Styling Criteria](/wiki/Article_Styling_Criteria)._
+
+To maintain consistency between all images on the osu!wiki, all images must adhere to the following image standards.
+
+## Metadata
+
+### Formats and quality
+
+Images used in the osu!wiki must be in JPG format at quality 8 (80). The PNG format can be used if the image contains transparency, while the GIF format can be used if the image contains an animation.
```

Comment:
code formats and change to ".xyz" where xyz is format name

Reply:
JPG, PNG, and GIF refers to the file's type, not the file's extenion
I'll change `JPG` to `JPEG` though

Reply:
never mind, I'll just use extension names